RELEASES.md

```diff
@@ -6,8 +6,87 @@ Unreleased.
 
 ### Added
 
+* Allow the MPL-2.0 and OpenSSL licenses in dependencies of wasmtime.
+  [#6136](https://github.com/bytecodealliance/wasmtime/pull/6136)
+
+* The initial integration between winch and wasmtime has landed behind a cargo
+  feature.
+  [#6119](https://github.com/bytecodealliance/wasmtime/pull/6119)
+
+* Add a bounds-checking optimization for dynamic memories and guard pages.
+  [#6031](https://github.com/bytecodealliance/wasmtime/pull/6031)
+
+* Add support for generating perf maps for simple perf profiling.
+  [#6030](https://github.com/bytecodealliance/wasmtime/pull/6030)
+
+* Add the `--default-values-unknown-imports` option to define unknown function
+  imports as functions that return the default value for their result type.
+  [#6010](https://github.com/bytecodealliance/wasmtime/pull/6010)
+
+* Introduce the `wasmtime-explorer` crate for investigating the compilation of
+  wasm modules.
```

Comment:
It might be worth mentioning that this is accessible through `wasmtime explore` as a CLI command perhaps?